wildfly/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheConfigurationService.java

```diff
@@ -71,8 +72,9 @@ protected ConfigurationBuilder getConfigurationBuilder() {
         builder.read(configuration);
         builder.template(configuration.isTemplate());
 
-       boolean jmxEnabled = SecurityActions.getCacheManagerConfiguration(dependencies.getCacheContainer()).statistics();
-       builder.jmxStatistics().enabled(jmxEnabled);
+        GlobalConfiguration globalConfiguration = SecurityActions.getCacheManagerConfiguration(dependencies.getCacheContainer());
+        builder.statistics().enabled(globalConfiguration.statistics());
+        // TODO [anistor] what about JMX ???
```

Comment:
WildFly was never into JMX, do nothing I guess /cc @tristantarrant

Reply:
WildFly does expose the DMR as JMX as well as expose the MBeanServer to subsystems. But this code is going to be dropped, so ignore.